src/main/java/com/hillel/zakushniak/lessons/lesson8/homework6/Main.java

```diff
@@ -0,0 +1,148 @@
+        System.out.println(Arrays.toString(twoSum(nums, target)));
+
+        nums = new int[]{3, 2, 4};
+        target = 6;
+        System.out.println(Arrays.toString(nums) + " target = " + target);
+        System.out.println(Arrays.toString(twoSum(nums, target)));
+
+        nums = new int[]{3, 3};
+        target = 6;
+        System.out.println(Arrays.toString(nums) + " target = " + target);
+        System.out.println(Arrays.toString(twoSum(nums, target)));
+
+//        System.out.println(Arrays.toString(twoSum(new int[]{3, 2, 4}, 6)));
+//        twoSum(new int[]{3, 2, 4}, 6);
+//        twoSum(new int[]{3, 3}, 6);
+
+
+    }
+
+    public static int[] twoSum(int[] nums, int target) {
+        int[] numsCloned = nums.clone();
+
+        Arrays.sort(numsCloned);
+        int i = 0;
+        int j = numsCloned.length - 1;
+
+        for (; i != j; ) {
+            if ((numsCloned[i] + numsCloned[j]) == target) {
+//                System.out.println("solution: " + i + " " + j);
+                break;
+            }
+            for (; j != i; ) {
+                if ((numsCloned[i] + numsCloned[j]) == target) {
+                    break;
+                }
+                if ((numsCloned[i] + numsCloned[j]) > target) {
+                    j--;
+                } else i++;
+            }
+        }
+        for (int k = 0; k < nums.length; k++) {
+            if (nums[k] == numsCloned[i]) {
+                i = k;
+                break;
+            }
+        }
+        for (int k = 0; k < nums.length; k++) {
+            if (nums[k] == numsCloned[j] && k != i) {
+                j = k;
+                break;
+            }
+        }
+        return new int[]{i, j};
+    }
+
+    public static boolean isElementsDistinct(int[] nums) {
+        Set<Integer> withoutDistincts = new HashSet<>();
+        for (int i = 0; i < nums.length; i++) {
+            if (withoutDistincts.contains(nums[i])) {
+                return true;
+            } else {
+                withoutDistincts.add(nums[i]);
+            }
+        }
+        return false;
+    }
+
+    //Time Limit Exceede
+//    public static boolean isElementsDistinct(int[] nums) {
+//        List<Integer> withoutDistincts = new ArrayList<>();
+//        for (int i = 0; i < nums.length; i++) {
+//            if (withoutDistincts.contains(nums[i])) {
+//                return true;
+//            } else {
+//                withoutDistincts.add(nums[i]);
+//            }
+//        }
+//        return false;
+//    }
+
+
+    //Time Limit Exceede
+//    public static boolean isElementsDistinct(int[] nums) {
+//        for (int i = 0; i < nums.length; i++) {
+//            int counter = 0;
+//            for (int j = 0; j < nums.length; j++) {
+//                if (nums[i] == nums[j]) counter++;
+//                if (counter > 1) return true;
+//            }
+//        }
+//        return false;
+//    }
+
```

Comment:
commenter code in pull request is forbidden

Reply:
is there any way to live it for me and not include to pull?